logger/logger.c

```diff
@@ -37,8 +37,9 @@ void logger_init (process_t process)
 		sprintf(process_str, "EXECUTOR");
 	} else if (process == NET_HANDLER) {
 		sprintf(process_str, "NET_HANDLER");
+	} else if (process == STUDENTAPI) {
```

Comment:
can you change it to `API` cus I'm too lazy to change my code

Reply:
We should try to be clear on what API we're talking about I think :P